test/battle_snake/snake_test.exs

```diff
@@ -160,20 +160,29 @@ defmodule BattleSnake.SnakeTest do
         url: "example.com",
         id: "1",
         taunt: "",
-        health_points: 100
+        health_points: 100,
+        color: "red",
+        head_url: "head.example.com",
       }
 
       expected = %{
         "name" => "snake",
         "coords" => [[0, 1]],
         "health_points" => 100,
         "taunt" => "",
-        "id" => "1"
+        "id" => "1",
       }
 
       actual = PoisonTesting.cast!(snake)
-
       assert expected == actual
+
+      # consumer encoding gets head_url and color
+      actual = Poison.decode!(Poison.encode!(snake, consumer: true))
+      assert actual == Map.merge(expected,
+                                 %{
+                                         "head_url" => "head.example.com",
+                                         "color" => "red",
+                                     })
```

Comment:
rather than coupling this with a prior definition, it's both easier to maintain and read if the entire contents of the structure were copied and pasted in place with the additions.